.github/PULL_REQUEST_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Pull Request for documentation
+about: Use this template for documentation changes
+title: ''
+labels: 'Documentation'
+assignees: ''
+
+---
+
+<!-- Thank you so much for your time taking to contribute, your work is appreciated! 😃 -->
+
+## Description
+<!-- A few sentences describing the overall goals of the pull request' s commits. -->
+
+*Documentation section involved*
+- [ ] User Guide
+- [ ] Developer Guide
+
+**Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
+- [ ] Tests for the changes have been added
+- [ ] Docs have been added / updated
+
+## Issue
```

Comment:
Please uses **Issues** with bullet points

Please move this just after **Description**